server/postgresQuery.js

```diff
@@ -10,85 +9,36 @@ const config = {
   port: 5432,
 };
 
-let imageCounter = 1;
-const getRandomImages = () => {
-  const results = [];
-  const limit = faker.random.number({ min: 4, max: 7 });
-
-  for (let j = 0; j < limit; j += 1) {
-    const imgURL = `https://aragorn-images.s3-us-west-2.amazonaws.com/${imageCounter}.jpg`;
-    results.push(imgURL);
-    imageCounter += 1;
-    if (imageCounter > 30) {
-      imageCounter = 1;
-    }
-  }
-  return results;
-};
-
-let count = 1;
-
-function createDatabase() {
-  pgtools.createdb(config, 'games', (err, res) => {
-    if (err) {
-      console.log('database already exists! - dropping database!');
-      pgtools.dropdb(config, 'games', (err, res) => {
-        if (err) {
-          console.error(err);
-          process.exit(-1);
-        }
-        count++;
-        createDatabase();
-      });
-    }
-    setTimeout(() => {
-      console.log('created database!');
-      createTable();
-    }, 3000);
-  });
-}
-if (count === 1) {
-  count++;
-  createDatabase();
-}
+const pool = new Pool({
+  user: 'postgres',
+  host: 'localhost',
+  database: 'games22',
+  password: 'samaung1',
+  port: 5432,
+});

```

Comment:
Same comment as in other files... this seems redundant with the above config variable, and could be abstracted into a single file that could be required into other files as needed rather than explicitly defined in every single file.

Reply:
yeah... I will work on that